### DISCUSSION TITLE
text2video

### DISCUSSION THREAD

**Discussion by almfahd41** [AIrjen/OneButtonPrompt#75] [Ideas]
hello , can you make your extensions work with this extensions ?

https://github.com/kabachuha/sd-webui-text2video

**Comment by AIrjen** [comment]
Hi, that doesn't seem worth doing.

There are some reasons, but the main one is:
- You can just manually copy paste a random generated prompt to it. You don't need me to build an entire integration flow for this.

There are some others reasons as well:
- text2video needs some more specific prompts describing an action. It needs specialized prompts to work correctly.
- It's not easy to integrate with other extensions, and maintaining it

There is one extension, which I DO want to integrate with, and that is infinite zoom:
https://github.com/v8hid/infinite-zoom-automatic1111-webui

I just need to get more time to actually do it in a proper way.
